ibm/data_source_ibm_cis_rate_limit.go

```diff
@@ -178,38 +179,40 @@ func dataSourceIBMCISRateLimit() *schema.Resource {
 }
 
 func dataSourceIBMCISRateLimitRead(d *schema.ResourceData, meta interface{}) error {
-	cisClient, err := meta.(ClientSession).CisAPI()
+	cisClient, err := meta.(ClientSession).CisRLClientSession()
 	if err != nil {
 		return err
 	}
 	cisID := d.Get("cis_id").(string)
-	zoneID, _, err := convertTftoCisTwoVar(d.Get("domain_id").(string))
```

Comment:
revert this change in the existing datasource

Reply:
done